test/module/irohad/multi_sig_transactions/state_test.cpp

```diff
@@ -129,17 +125,20 @@ TEST(StateTest, DifferentSignaturesUnionTest) {
   state1 +=
       addSignatures(makeTestBatch(txBuilder(3)), 0, makeSignature("3", "3"));
 
-  ASSERT_EQ(3, state1.getBatches().size());
+  ASSERT_EQ(3, state1.batchesQuantity());
 
-  auto state2 = MstState::empty(mst_state_log_, completer_);
+  auto state2 = MstState::empty(completer_, kMstStateTxLimit, mst_state_log_);
   state2 +=
       addSignatures(makeTestBatch(txBuilder(4)), 0, makeSignature("4", "4"));
   state2 +=
       addSignatures(makeTestBatch(txBuilder(5)), 0, makeSignature("5", "5"));
-  ASSERT_EQ(2, state2.getBatches().size());
+  ASSERT_EQ(2, state2.batchesQuantity());
 
+  ASSERT_LE(state1.transactionsQuantity() + state2.transactionsQuantity(),
```

Comment:
I cannot agree because here we check against the transaction limit, and in the last line against a number not related to it. Suppose someone changes `kMstStateTxLimit`. Will he be responsible for checking every magic number in all related tests?